addons/resource/models/resource.py

```diff
@@ -749,6 +757,7 @@ class ResourceCalendarLeaves(models.Model):
     resource_id = fields.Many2one(
         "resource.resource", 'Resource',
         help="If empty, this is a generic holiday for the company. If a resource is set, the holiday/leave is only for this resource")
+    time_type = fields.Selection([('leave', 'Leave'), ('other', 'Other')], default='leave')
```

Comment:
I would add an help explaining a bit the purpose and what we could do with this field.